helpers/make_template.py

```diff
@@ -50,15 +53,16 @@ def main (options_file, template_file):
                     entered_restart_section=True
 
                 key = (l.split("=")[0]).strip().lower()
-                if key == "restart_file":
-                    if verbose: print("Writing restart_file line")
-                    tmpl.write('    restart_file="__RESTART_FILE__"\n')
-                elif key == "restart_date":
+                ## as long as the default (i.e. restart_file="restart/icar_rst_" ) is used, no need to change.
```

Comment:
but what if someone wants to use different file names? Is there a reason this section has to be removed?

Reply:
it was breaking the code. Need to go through this again to see how it can be more robust. Does anyone use a different structure though?

Reply:
There is a restart_file=.. line in &output_list and one in &restart_info. THey  both get overwritten with this line, which is not what we want.